gengokit/handler/hooks.go

```diff
@@ -0,0 +1,31 @@
+package handler
+
+import (
+	"io"
+	"strings"
+
+	"github.com/TuneLab/go-truss/gengokit"
+)
+
+const HookPath = "NAME-service/handlers/server/hooks.gotemplate"
+
+// NewHook returns a new HookRender
+func NewHook(prev io.Reader) gengokit.Renderable {
+	return &HookRender{
+		prev: prev,
+	}
+}
+
+type HookRender struct {
+	prev io.Reader
+}
+
+// Render will return the existing file if it exists, otherwise it will return
+// a brand new copy from the template.
+func (h *HookRender) Render(_ string, _ *gengokit.Data) (io.Reader, error) {
+	if h.prev == nil {
+		return strings.NewReader(hookTempl), nil
+	} else {
```

Comment:
In Go, if you return inside an if, the preferred style is to omit the `else`, since it's not necessary. Golint/gometalinter will catch stuff like this.